CONTRIBUTING.md

```diff
@@ -10,6 +10,7 @@
 * Code the algorithm/DS following the styleguide defined below.
 * Send a PR. 
 * Be sure to not include any compiled binaries in the patch.
+* While sending a PR make sure you follow one PR per issue rule.
```

Comment:
Your reason for starting that issue was "I think sending one PR per issue would be nice rather than adding 2 or more issues in a single PR, what are your views over this?" . 

So it should be `one issue per PR`.